Sources/UI/Web View/WebViewController.swift

```diff
@@ -37,6 +37,18 @@ open class WebViewController: UIViewController, WKNavigationDelegate {
         }
     }
     
+    override open func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        setUIDocumentMenuViewControllerSoureViewsIfNeeded(viewControllerToPresent)
+        super.present(viewControllerToPresent, animated: flag, completion: completion)
+    }
+    
+    func setUIDocumentMenuViewControllerSoureViewsIfNeeded(_ viewControllerToPresent: UIViewController) {
+        if #available(iOS 13, *), viewControllerToPresent is UIDocumentMenuViewController && UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
Is it needed only for iPhone devices?

Reply:
I bet it's required for ipads and big iphones. 
Also, do you need check `viewControllerToPresent is UIDocumentMenuViewController`? As far as I know it will be required for any modally presented view controller

Reply:
It's only required for phones, iPads are correctly setting up the controller. (Tested all sizes)
It's only required for `UIDocumentMenuViewController` (check SO link in description)